source/docs/v3/integrations/sail.blade.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Laravel Sail integration
+extends: _layouts.documentation
+section: content
+---
+
+# Laravel Sail {#sail}
+
+## Creating tenants
+
+Before creating any tenants you have to ensure that the `sail` user has the necessary permissions.
+
+To do so either login to the mysql shell or via any client using root user credentials and run the following queries:
+
+```bash
+GRANT ALL PRIVILEGES on *.* to 'sail'@'%';
+FLUSH PRIVILEGES;
+```
+
+### Specifying Database Credentials {#specifying-database-credentials}
+
+While working with sail, it offers a database user `sail` which just have the privileges for just the one database created at the inception of the project (central database in our tenancy case). This user does not have the privileges to create new users or `GRANT OPTION` privilege, which is used to grant permissions to other users.
+
+If you want to use custom users (one user for each respective database), you will need `GRANT OPTION` grant on your central database user (`sail` in this case), otherwise you cannot grant permissions to the newly created user and will result in an error.
+
+To avoid this failure, run the following SQL from root user:
+
+```sql
+GRANT GRANT OPTION on central_database.* to 'sail'@'%';
+FLUSH PRIVILEGES;
+```
```

Comment:
I don't understand the point of this section. This looks like a reiteration of the first section?

Reply:
You *can* have one user per tenant, the package has a feature for that, but I don't see what this section does differently compared to the previous section.

Reply:
I was speaking of permissions, specifically this `GRANT OPTION`. Okay, ideally this is not the right place then. Sail has nothing to do with this grant. I will just remove this section then.